### PR TITLE
Update helper.js

### PR DESCRIPTION
**Pull request by catmandx**
Changed download button to a tag so that it's possible to right click and copy link.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -15,7 +15,7 @@ import (
 
 // command-line banner information
 var (
-	appVersion = "development"
+	appVersion = "v0.2.7-1"
```
Comment (ngoduykhanh) [inline main.go:18]: Why setting this version?

File: prepare_assets.sh [hunk 2]
```diff
@@ -4,7 +4,7 @@ set -e
 DIR=$(dirname "$0")
 
 # install node modules
-yarn install --pure-lockfile --production
+# yarn install --pure-lockfile --production
```
Comment (ngoduykhanh) [inline prepare_assets.sh:7]: Why do we remove this step?

File: yarn.lock [hunk 3]
```diff
@@ -3,139 +3,139 @@
```
Comment (ngoduykhanh) [inline yarn.lock:3]: I don't think we need to update the this file as the package file remains the same

File: templates/base.html [hunk 4]
```diff
@@ -222,7 +222,7 @@ <h1>{{template "page_title" .}}</h1>
             <div class="float-right d-none d-sm-block">
                 <b>Version</b> {{ .appVersion }}
             </div>
-            <strong>Copyright &copy; 2020 <a href="https://github.com/ngoduykhanh/wireguard-ui">Wireguard UI</a>.</strong> All rights
+            <strong>Copyright &copy; 2021 <a href="https://github.com/catmandx/wireguard-ui">Wireguard UI</a>.</strong> All rights
```
Comment (ngoduykhanh) [inline templates/base.html:225]: Why do you replace the original project link? :)

**Review by ngoduykhanh — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 5]
```diff
@@ -21,3 +21,5 @@ rice-box.go
 # IDEs
 .vscode
 .idea
+package-lock.json
+yarn.lock
```
Comment (ngoduykhanh) [inline .gitignore:25]: They should be in the version control

**Comment by catmandx** [comment]
Hey, I didn't know that additional commits would show up here with the original PR commit. Sorry for the inconvenience.

**Comment by catmandx** [comment]
I have reverted the problematic commits, please review the changes to the download functionality.

**Review by ngoduykhanh — APPROVED** [review]
LGTM now. Merging your PR. Thanks for your contribution!

File: handler/routes.go [hunk 6]
```diff
@@ -346,7 +346,7 @@ func DownloadClient() echo.HandlerFunc {
 		reader := strings.NewReader(config)
 
 		// set response header for downloading
-		c.Response().Header().Set(echo.HeaderContentDisposition, "attachment; filename=wg0.conf")
+		c.Response().Header().Set(echo.HeaderContentDisposition, fmt.Sprintf("attachment; filename=%s.conf", clientData.Client.Name))
```
Comment (varhub) [inline handler/routes.go:349]: this tiny change made my day. Thank you.

**Pull request merged**